appinventor/codelabs/talk-to-me/index.html

```diff
@@ -0,0 +1,163 @@
+          <h2>talk-to-me-1</h2>
+          <p><b>To receive this checkoff, you must</b></p>
+          <ul class="checklist">
+            <li>Have a button that says "Talk To Me"</li>
+            <li>Describe what the Properties section of each component does</li>
+            <li>Explain why the TextToSpeech component does not appear on the viewer screen</li>
+          </ul>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Connect your phone">
+        <p>At the top of the App Inventor website, find <b>Connect</b>, and select <b>AI Companion</b> from the drop down menu.</p>
+        <img src="img/ai_companion.png">
+        <p>Open the AI2 Companion App on your phone. Select <b>scan QR code</b> and use your phone to scan the code that appears on your computer. Click <b>connect with code</b>.</p>
+        <img style="max-height: 200px" src="img/companion_app.png">
+        <p>Do you see your app on your phone? App Inventor lets you test your app on your phone as you build it on the computer!</p>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Make your app talk">
+        <p>In <b>Blocks View</b>, open the <b>Button</b> drawer and drag the <code>when Button1.Click</code> block into the workspace.</p>
+        <p>Open the <b>TextToSpeech</b> drawer and plug in the <code>call TextToSpeech1.Speak</code> block into the <code>when Button1.Click</code> <b>Event Handler</b>.</p>
+        <img src="img/button_block.png">
+        <p>Next, open the <b>Text</b> drawer and drag a <b>text</b> block into the <code>message</code> socket. Click the <b>Text</b> block and type "Congratulations! You've made your first app."</p>
+        <img src="img/text_block.png">
+        <p>Test your app by clicking the button on your phone.</p>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Checkoff #2">
+        <aside class="special">
+          <h2>talk-to-me-2</h2>
+          <p><b>To recieve this checkoff, you must</b></p>
+          <ul class="checklist">
+            <li>Show your app talking when you press the button</li>
+            <li>Describe how the <code>when Button1.Click</code> block and the <code>call TextToSpeech1.Speak</code> block work together.</li>
+            <li>Explain why the <code>call TextToSpeech1.Speak</code> block is inside of the <code>when Button1.Click</code> block.</li>
+          </ul>
+        </aside>
+
+      </google-codelab-step>
+
+      <google-codelab-step label="Congratulations!">
+        <p>You have made your first app using App Inventor.</p>
+        <h3 class="checklist"><strong>What we&#39;ve covered</strong></h3>
+        <ul class="checklist">
+          <li>Use the <b>Designer View</b> to create the user interface.</li>
+          <li>Use the <b>Blocks View</b> to add functionality to the app.</li>
+          <li>Use the <b>Button</b> and <b>TextToSpeech</b> components.</li>
+          <li>Use <b>Event Handlers</b>.</li>
```

Comment:
Could you add an explanation of what an event handler is in one of the previous steps?